src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.ts

```diff
@@ -174,15 +213,32 @@ export const AzureComputeModalBase = ({
 
   const doesRuntimeExist = () => !!currentRuntimeDetails;
 
+  interface CommonButtonProps {
```

Comment:
this seems like it should live elsewhere, perhaps in the components package?

Reply:
Maybe rename to ActionButtonProps

Reply:
done